ext/EnzymeStaticArraysExt.jl

```diff
@@ -21,4 +21,11 @@ end
     end
 end
 
+
```

Comment:
since this is completely commented out, let's remove it.

Also how it is api breaking. if the change is now that a staticarray is returned instead of an array, I'm fine with that change (the tuple one, people already use quite heavily so I'd want to keep that in a breaking change to make it easy for folks).

Reply:
The issue here is entirely with the `gradient` methods that are currently returning `Tuple` instead of array.  If this is uncommented this will return `StaticArray` in cases where the `Array` equivalent would return `Tuple` (this is a good example of why I think ultimately the breaking change is called for).

Are you sure this should be removed?  If this PR to get merged, it seems like a clear TODO that this should be handled at some point and having it here would make it a lot easier to keep track of.

Reply:
I suppose that's fair. My concern here is just that commented out code is never run, and thus can quickly get out of data (and confuse folks).

I'm okay with leaving it here, or putting it in an issue with the NOTE for todo, or whatever you prefer

Reply:
Well, I suppose we are going to need to tackle the subject of how exactly breaking changes will actually work.  (Of course this could be uncommented then.)  By this point I assume from your interest in these PR's that you agree that there are some (mildly) breaking changes needed here for sake of consistency, but I don't know what the time frame for that would look like.  Presumably when we get through these PR's we can have a more detailed (and visible) conversation about that.